test/e2e/flowaggregator_test.go

```diff
@@ -300,14 +301,24 @@ func testHelper(t *testing.T, data *TestData, isIPv6 bool) {
 	// and their flow information is exported as IPFIX flow records.
 	// K8s network policies are being tested here.
 	t.Run("IntraNodeFlows", func(t *testing.T) {
-		checkIntraNodeFlows(t, data, podAIPs, podBIPs, isIPv6)
+		label := "IntraNodeFlows"
+		rc, _, _, err := data.RunCommandOnNode(controlPlaneNodeName(), fmt.Sprintf("kubectl label pods -l app=iperf -n %s targetLabel=%s --overwrite", data.testNamespace, label))
```

Comment:
it's always an anti-pattern to use `RunCommandOnNode` in the e2e tests, and any new usage should be discouraged

you should be using the K8s API here to mutate Pod labels

Reply:
Fixed. Thanks.